Brokerages/Zerodha/ZerodhaBrokerage.cs

```diff
@@ -357,32 +358,6 @@ private void OnOrderUpdate(Messages.Order orderUpdate)
         }
         
 
-        /// <summary>
-        /// Lock the streaming processing while we're sending orders as sometimes they fill before the REST call returns.
-        /// </summary>
-        public void LockStream()
-        {
-            Log.Trace("ZerodhaBrokerage.Messaging.LockStream(): Locking Stream");
-            _streamLocked = true;
-        }
-
-        /// <summary>
-        /// Unlock stream and process all backed up messages.
-        /// </summary>
-        public void UnlockStream()
-        {
-            Log.Trace("ZerodhaBrokerage.Messaging.UnlockStream(): Processing Backlog...");
-            while (_messageBuffer.Any())
-            {
-                MessageData e;
-                _messageBuffer.TryDequeue(out e);
```

Comment:
`_messageBuffer` can be removed now? `LastHeartbeatUtcTime, ApiSecret, ChannelList` are not used, should we remove or used them?

Reply:
`ApiSecret` is being used to get `Kite `object to deal with zerodha api
Rest all we can remove